mondey_backend/src/mondey_backend/routers/users.py

```diff
@@ -63,6 +65,13 @@ def create_child(
         current_active_user: CurrentActiveUserDep,
         child: ChildCreate,
     ):
+        current_month = int(datetime.now().strftime("%m"))  # 01 for Jan
+        current_year = int(datetime.now().strftime("%Y"))
+        if (
+            child.birth_month > current_month and child.birth_year >= current_year
+        ) or child.birth_year >= current_year:
```

Comment:
here we can avoid doing the logic (which is easy to get wrong) ourselves by comparing the dates:
```suggestion
        if datetime(child.birth_year,child.birth_month,1) > datetime.now():
```

Reply:
Sounds good